core/audits/critical-request-chains.js

```diff
@@ -199,7 +199,7 @@ class CriticalRequestChains extends Audit {
       walk(initialNavChildren, 0);
     }
 
-    const longestChain = CriticalRequestChains._getLongestChain(flattenedChains);
+    const longestChain = CriticalRequestChains._getLongestChain(chains);
```

Comment:
AFAICT this never made sense to run on `flattenedChains`, since it recomputes the `durations` anyways, but it previously worked either way. Now it just needs to drop the `* 1000` since request times are in milliseconds after #14567